htdocs/commande/class/commande.class.php

```diff
@@ -3163,6 +3181,18 @@ public function updateline($rowid, $desc, $pu, $qty, $remise_percent, $txtva, $t
 				$this->line->rang = $rangmax + 1;
 			}
 
+			if (getDolGlobalString('PRODUCT_USE_CUSTOMER_PACKAGING')) {
+				if ($qty < $this->line->packaging) {
+					$qty = $this->line->packaging;
+				} else {
+					if (!empty($this->line->packaging) && ($qty % $this->line->packaging) > 0) {
```

Comment:
I think there is an issue when using % on floats.

```
$ php -r "print 120.1 %1.1;"

Deprecated: Implicit conversion from float 120.1 to int loses precision in Command line code on line 1
```

`fmod` is an alternatvie.